src/amp-core-service.js

```diff
@@ -28,4 +29,5 @@ export function installCoreServices(window) {
   installViewerService(window);
   installViewportService(window);
   installHistoryService(window);
+  installVsyncService(window);
```

Comment:
Seems like vsync should be one of the first to be installed, or maybe right after viewer.